secretmanager/snippets/regional_samples/snippets_test.py

```diff
@@ -104,6 +109,27 @@ def secret_id(
         # Secret was already deleted, probably in the test
         print(f"Secret {secret_id} was not found.")
 
+@pytest.fixture()
+def regional_secret_version(
+    regional_client: secretmanager_v1.SecretManagerServiceClient,
+    regional_secret: Tuple[str, str, str, str],
+) -> Iterator[Tuple[str, str, str, str, str]]:
+    project_id, location_id, secret_id, _ = regional_secret
+
+    print(f"adding secret version to {secret_id}")
+    parent = f"projects/{project_id}/locations/{location_id}/secrets/{secret_id}"
+    payload = b"hello world!"
+    time.sleep(5)
+    version = regional_client.add_secret_version(
+        request={"parent": parent, "payload": {"data": payload}}
+    )
```

Comment:
```suggestion
    time.sleep(5)
    version = regional_client.add_secret_version(
        request={
            "parent": f"projects/{project_id}/locations/{location_id}/secrets/{secret_id}",
            "payload": {"data": b"hello world!"}
        },
    )
```
For tests, prefer using direct values than masking them in variables, unless they are reused in various test cases.